### ISSUE TITLE
Cloud Scale Unit manifest does not allow for commas in the publisher name

### ISSUE THREAD

**Issue by rphillips-newestech** [microsoft/Dynamics365Commerce.ScaleUnit#6]
When a company's signing certificate includes commas in the publisher information, the relevant parts have to be surrounded by quotation marks so that those commas are not seen as delimiters.

When the [ScaleUnit](https://github.com/microsoft/Dynamics365Commerce.ScaleUnit/tree/release/9.34/src/ScaleUnitSample/ScaleUnit)/.../CloudScaleUnitExtensionPackage.zip is produced, the double quotes are not correctly written to the base manifest.json file. For example, the output file looks like this.

```
{
    "sdkVersion": "...",
    "customPackageName": "...",
    "customPackageDescription": "...",
    "customPackagePublisher": "CN="My Test Company, INC", O="My Test Company, INC", STREET=123 N Main St., L=Schenectady, S=NY, PostalCode=12345, C=US",
    "customPackageVersion": "..."
}
```
When `"customPackagePublisher"` should have value `"CN=\"My Test Company, INC\", O=\"My Test Company, INC\", STREET=123 N Main St., L=Schenectady, S=NY, PostalCode=12345, C=US"`

The Cloud Scale Unit fails on load because of the malformed JSON. As a workaround I am unzipping this file, updating the manifest manually, and then re-zipping the file.

**Comment by Pikhulya** [comment]
@rphillips-newestech, this has been fixed in 10.0.27.

**Comment by rphillips-newestech** [comment]
Understood. Thanks!
